contracts/lootbox-v1/contracts/Lootbox.sol

```diff
@@ -0,0 +1,162 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Highly recommend to add ascii art here.
https://patorjk.com/software/taag/#p=display&f=3D-ASCII&t=Ubiquity

Reply:
lets gooooo

```

 ___  ___  ________  ___  ________  ___  ___  ___  _________    ___    ___ 
|\  \|\  \|\   __  \|\  \|\   __  \|\  \|\  \|\  \|\___   ___\ |\  \  /  /|
\ \  \\\  \ \  \|\ /\ \  \ \  \|\  \ \  \\\  \ \  \|___ \  \_| \ \  \/  / /
 \ \  \\\  \ \   __  \ \  \ \  \\\  \ \  \\\  \ \  \   \ \  \   \ \    / / 
  \ \  \\\  \ \  \|\  \ \  \ \  \\\  \ \  \\\  \ \  \   \ \  \   \/  /  /  
   \ \_______\ \_______\ \__\ \_____  \ \_______\ \__\   \ \__\__/  / /    
    \|_______|\|_______|\|__|\|___| \__\|_______|\|__|    \|__|\___/ /     
                                   \|__|                      \|___|/      
                                                                           
                                                                           

```